metadata.rb

```diff
@@ -14,7 +14,7 @@
 supports 'scientific'
 supports 'oracle'
 
-depends  'python'
+depends  'poise-python', '~> 1.5.1'
```

Comment:
We shouldn't pin community cookbook to minor versions unless absolutely necessary. Something like `>= 1.5` would work.